roles/aap_ocp_install/tasks/install-eda.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Create EDA instance
```

Comment:
There should be a task to create a namespace if the defined namespace doesn't exist. See the first task in the `install-controller.yml` file for an example. You should be able to copy that task and change the variable names. (I realized in reviewing this PR that the hub install tasks don't do this either so I'll be fixing that with an upcoming PR).

Reply:
Done!